lib/adobe/campaign/base.rb

```diff
@@ -13,6 +13,7 @@ def all
         end
 
         def find(search_text)
+          search_text = ERB::Util.url_encode(search_text)
```

Comment:
I don't see `url_encode` in Rails 6.1, which mpdx-api still uses (but not for long), as well as missionhub-api (does that matter?)

Reply:
I'm confused. It's in ruby - https://msp-greg.github.io/ruby_3_1/erb/ERB/Util.html#url_encode-class_method

Reply:
I'm planning on deploying Rails 7 for mpdx on Nov 18.

I don't think we care about missionhub anymore.

Reply:
> I'm confused. It's in ruby

OIC; maybe it migrated from Rails to Ruby.